src/main/java/com/eternalcode/discordapp/review/GitHubReviewService.java

```diff
@@ -184,14 +184,37 @@ public void deleteMergedPullRequests(JDA jda) {
         }
     }
 
-    public void addUserToSystem(GitHubReviewUser gitHubReviewUser) {
+    public String addUserToSystem(GitHubReviewUser gitHubReviewUser) {
+        if (this.isUserExist(gitHubReviewUser)) {
+            return "User already exists";
+        }
+
         this.discordAppConfig.reviewSystem.reviewers.add(gitHubReviewUser);
         this.configManager.save(this.discordAppConfig);
+
+        return "User added";
     }
 
-    public void removeUserFromSystem(Long discordId) {
+    public String removeUserFromSystem(Long discordId) {
```

Comment:
Nie wiem czy używanie typu `String` jest tu dobre, bardziej bym poszedł w `Result`, chyba że wolisz minimalizm to `int` po status code'ach?